sys-libs/liburing/liburing-2.3-r2.ebuild

```
@@ -17,7 +17,7 @@ fi
 LICENSE="MIT"
 SLOT="0/2" # liburing.so major version
 
-IUSE="static-libs"
+IUSE="examples static-libs test"
 # fsync test hangs forever
 RESTRICT="test"
```

To tie USE=test to FEATURES=test:
```suggestion
RESTRICT="test !test? ( test )"
```

---

I'm no expert, is it ```test``` **AND** ``!test? ( test )```?

---

Yeah, in this case, it genuinely is both, because tests are restricted due to them being broken, but if they weren't, we still want that latter !test ... bit.

That is, when tests work again, someone can just drop the first bit, and everything is ready.